### DISCUSSION TITLE
Is it possible to embed a Jupyter notebook cell in a second notebook rendered in revealjs?

### DISCUSSION THREAD

**Discussion by lebedov** [quarto-dev/quarto-cli#6519] [Q&A] [answered]
Is it possible to embed a cell in one Jupyter notebook within another notebook such that rendering the latter as a revealjs presentation will cause the former to be included? I tried creating a `sub_notebook.ipynb` file with a single code cell containing
```python
#| label: fig_1
import matplotlib.pyplot as plt
import numpy as np
plt.scatter(np.random.rand(10), np.random.rand(10));
```
and a `main_notebook.ipynb` file with a raw cell containing
```yaml
---
title: Main Notebook
format: revealjs
---
```
followed by a second Markdown cell containing
```markdown
{{< embed sub_notebook.ipynb#fig_1 >}}
```
I then tried rendering `main_notebook.ipynb` into HTML after generating the plot in `sub_notebook.ipynb` and saving that notebook; however, the cell from `sub_notebook.ipynb` did not appear in the rendering of the latter. 

The context for the above is trying to pull in select results from several Jupyter notebooks into one notebook used to present the results. Yes, I know one can create a raw Markdown file that pulls in the results from several notebooks, but it would be useful to do the same with a notebook so that the resulting presentation could also use content local to that notebook.
I'm using quarto 1.3.450 on MacOS 13.5.

**Comment by dragonstyle** [comment, marked as answer]
I just took this for a test run on Quarto 1.4.302 (pre-release) <https://quarto.org/docs/download/prerelease> and everything appeared to be working. We have been making a number of improvements to `embed` in Quarto 1.4, so it is definitely possible that this changed between Quarto 1.3 and 1.4. 

If you still see problems - it would be useful to have a sample repo / notebooks to try. I tried to reproduce what you have above a as closely as possible, but sometimes there is a subtle difference that results in the issue not reproducing.

**Reply by lebedov** [reply]
Confirmed - it does indeed work with Quarto 1.4.302, although it seems that no link to the source notebook is added. @dragonstyle Is any link being generated on your end?

**Reply by dragonstyle** [reply]
No - we currently don't generate notebook previews for RevealJs (or books).

**Reply by lebedov** [reply]
No matter - the core embedding functionality is sufficient for my purposes. Thanks for the quick response!
